src/components/Questions/Body.vue

```diff
@@ -298,9 +321,25 @@ export default defineComponent({
     }
 
     function preventKeypressIfApplicable(event: KeyboardEvent) {
-      // checks if character limit is reached in case it is set
-      if (!hasCharLimit.value) return;
-      if (!charactersLeft.value) event.preventDefault();
+      if (isQuestionTypeSubjective.value) {
+        // checks if character limit is reached in case it is set
+        if (!hasCharLimit.value) return;
+        if (!charactersLeft.value) event.preventDefault();
+      }
+      if (isQuestionTypeFloatNumerical.value) {
+        // console.log($event.keyCode); //keyCodes value
+        const keyCode = event.keyCode ? event.keyCode : event.which;
+        if ((keyCode < 48 || keyCode > 57) && keyCode !== 46) {
```

Comment:
So we are accepting answers like .46? Not forcing the user to type 0.46?

Reply:
yes